app/controllers/MessagesController.scala

```diff
@@ -79,27 +79,23 @@ class MessagesController @Inject()(
                   request.body
                 ) match {
                 case Right(message) =>
-                  StatusTransition.targetStatus(request.departure.status, MessageReceivedEvent.DeclarationCancellationRequest) match {
```

Comment:
Does removing the call to targetStatus mean we now have different logic for a cancellation request?

Reply:
The entire DepartureStatus has been removed from the mongodb and is instead calculated from the messages. In the long run we won't even be sent this value